diagnostic.md

```diff
@@ -35,7 +37,12 @@ Record your responses inside the fenced code blocks below each question.
     ```
 
     ```md
-    <!-- your response here -->
+    The first route definition has a nested route, so the parent url will be at the URL
+    `/products` while the child (for the first product) will be at `products/1`
+    or something similar.
+
+    The second is a single route that will have the url `products/1` for the first
+    produc.
```

Comment:
👍 
More explanation:
The first `'product'` route definition is nested under the `'products'`
definition, whereas the second `'product'` route definition is a sibling of
the `'products'` definition.

Nested route definitions, like the first `'product'` route above, require
a nested file structure, such that Route and Template files for `'product'`
are expected to be found at `app/products/product`. For sibling routes,
folders are siblings as well.

The first `'product'` route definition instructs Ember to render
`app/products/product/template.hbs` in the context (the `outlet`) of the
parent `app/products/template.hbs`. The second `'product'` route definition
instructs Ember to render `app/product/template.hbs` in the context of the
application template.

These two route definitions do **not** have different paths.